frb_example/pure_dart/rust/build.rs

```diff
@@ -22,6 +22,18 @@ fn main() {
         // for other options use defaults
         ..Default::default()
     };
+
+    if cfg!(feature = "c-output") {
+        raw_opts.c_output = Some(vec!["./c_output_path/c_output.h".into()]);
+    }
+
+    if cfg!(feature = "extra-c-output-path") {
+        raw_opts.extra_c_output_path = Some(vec![
+            // For test, the below 2 paths format are made a little different
+            "./extra_c_output_path_1/".into(),
+            "extra_c_output_path_2".into(),
+        ]);
+    }
```

Comment:
Ready to merge! Given that we have thorough unit tests, maybe we can remove these in pure_dart example (indeed I have not required this one - I originally meant unit tests), and only remain such end-to-end tests in pure_dart_multi. Because pure_dart is already quite huge, and adding such tests will make generated content tripled.

Reply:
Done since d84c77b18632b1eabc4957cb2a3d203531c89baa.